### PR TITLE
API-5465 Clarify when thread_id is present in chat-user-added/removed.

### PR DESCRIPTION
**Pull request by gluszczyk**
https://livechatinc.atlassian.net/browse/API-5465

### CODE REVIEW

File: content/messaging/agent-chat-api/rtm-reference/index.mdx [hunk 1]
```diff
@@ -4599,6 +4599,7 @@ This push can be emitted with `user.present` set to `false` when a user writes t
 | Object      | Notes                                |
 | ----------- | ------------------------------------ |
 | `user_type` | Possible values: `agent`, `customer` |
+| `thread_id` | Empty if a user added to an inactive chat. |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/rtm-reference/index.mdx]: In such a case, thread_id will be an empty string?

Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/rtm-reference/index.mdx]:
```suggestion
| `thread_id` | Empty if a user was added to an inactive chat. |
```

File: content/messaging/agent-chat-api/v3.2/rtm-reference/index.mdx [hunk 2]
```diff
@@ -4598,6 +4598,7 @@ This push can be emitted with `user.present` set to `false` when a user writes t
 | Object      | Notes                                |
 | ----------- | ------------------------------------ |
 | `user_type` | Possible values: `agent`, `customer` |
+| `thread_id` | Present when a user added to an active chat. |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/rtm-reference/index.mdx]:
```suggestion
| `thread_id` | Present when a user was added to an active chat. |
```

File: content/messaging/customer-chat-api/rtm-reference/index.mdx [hunk 3]
```diff
@@ -3118,6 +3118,7 @@ This push can be emitted with `user.present` set to `false` when a user writes t
 | Object      | Notes                                |
 | ----------- | ------------------------------------ |
 | `user_type` | Possible values: `agent`, `customer` |
+| `thread_id` | Empty if a user added to an inactive chat. |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/rtm-reference/index.mdx]:
```suggestion
| `thread_id` | Empty if a user was added to an inactive chat. |
```

File: content/messaging/customer-chat-api/v3.2/rtm-reference/index.mdx [hunk 4]
```diff
@@ -3336,6 +3336,7 @@ This push can be emitted with `user.present` set to `false` when a user writes t
 | Object      | Notes                                |
 | ----------- | ------------------------------------ |
 | `user_type` | Possible values: `agent`, `customer` |
+| `thread_id` | Present when a user added to an active chat. |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.2/rtm-reference/index.mdx]:
```suggestion
| `thread_id` | Present when a user was added to an active chat. |
```

File: content/messaging/agent-chat-api/rtm-reference/index.mdx (hunk 1 above)
Comment (gluszczyk) [reply content/messaging/agent-chat-api/rtm-reference/index.mdx]: yes

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/rtm-reference/index.mdx]: ok 👍🏻

**Pull request merged**